src/Insect/Parser.purs

```diff
@@ -224,8 +224,8 @@ normalUnitDict = Dictionary
   , Q.week ==> ["weeks", "week"]
   , Q.month ==> ["months", "month"]
   , Q.year ==> ["years", "year"]
-  , Q.gram ==> ["grams", "gram", "g"]
-  , Q.meter ==> ["meters", "meter", "m"]
+  , Q.gram ==> ["grams", "gram", "grammes", "gramme", "g"]
```

Comment:
Unfortunately, the order of these words is important. In order for the parser to work correctly, it should be

``` purs
["grammes", "gramme", "grams", "gram", "g"]
```

Otherwise, the parser will successfully parse "gram" first and then find "mes", which cannot be interpreted.